query-specifications/bi-read-02.yaml

```diff
@@ -3,7 +3,7 @@ operation: read
 number: 2
 title: Top tags for country, age, gender, time
 description: |
-  Select all *Messages* (Posts & Comments) created between `date1`-`date2` (inclusive) by *Persons* located in `country1` or `country2`.
+  Select all *Messages* created between `date1`-`date2` (inclusive) by *Persons* located in `country1` or `country2`.
```

Comment:
Can we make the interval specification consistent in all queries? I think that in other queries (e.g. look query 14, we use the typical interval notation (witn "[" and "(" to denote close and open respectively, if I am not wrong)

Reply:
Pushed `f671abd` to fix this.